Documentation/architecture.rst

```diff
@@ -263,18 +264,18 @@ Session Affinity         node             64k             Max 64k affinities fro
 For some BPF maps, the upper capacity limit can be overridden using command
 line options for ``cilium-agent``. A given capacity can be set using
 ``--bpf-ct-global-tcp-max``, ``--bpf-ct-global-any-max``,
-``--bpf-nat-global-max``, ``--bpf-policy-map-max``, and
-``--bpf-fragments-map-max``.
-
-Using ``--bpf-map-dynamic-size-ratio`` the upper capacity limits of the
-connection tracking, NAT, and policy maps are determined at agent startup based
-on the given ratio of the total system memory. For example a given ratio of 0.03
-leads to 3% of the total system memory to be used for these maps.
-
-This flag sets the ratio of total system memory to use for dynamic size of the
-BPF maps that consume most memory in the system: ``cilium_ct_{4,6}_global``,
-``cilium_ct_{4,6}_any``, ``cilium_nodeport_neigh{4,6}``,
-``cilium_snat_v{4,6}_external`` and ``cilium_lb{4,6}_reverse_sk``.
+``--bpf-nat-global-max``, ``--bpf-neigh-global-max``, ``--bpf-policy-map-max``,
+and ``--bpf-fragments-map-max``.
+
+Using the ``--bpf-map-dynamic-size-ratio`` flag, the upper capacity limits of
+several large BPF maps are determined at agent startup based on the given ratio
+of the total system memory. For example, a given ratio of 0.025 leads to 0.25% of
+the total system memory to be used for these maps.
```

Comment:
I thought the example was just an example so 0.03% is not wrong :smile:

Reply:
:smile: true, just thought it would be nice to be consistent